usr/share/rear/output/RAWDISK/Linux-i386/270_create_grub2_efi_bootloader.sh

```diff
@@ -39,5 +49,5 @@ EOF
 
 # Create a Grub 2 EFI core image
 local grub_modules=( part_gpt fat normal configfile linux video all_video )
-grub-mkimage -O x86_64-efi -o "$efi_boot_directory/BOOTX64.efi" -p "/EFI/BOOT" "${grub_modules[@]}"
+$grub_name"-mkimage" -O x86_64-efi -o "$efi_boot_directory/BOOTX64.efi" -p "/EFI/BOOT" "${grub_modules[@]}"
```

Comment:
Only a side note FYI:
You don't need the double quotes because `-` is not an allowed character
in a bash variable name so that bash recognizes the variable name is
only up to the `-` character, e.g as in
```
# ( set -x ; grub_name=grub2 ; type -a $grub_name-mkimage )
+ grub_name=grub2
+ type -a grub2-mkimage
grub2-mkimage is /usr/bin/grub2-mkimage
```

Reply:
right! didn't know, thanks for the info!